lib/RoutingList/hooks/useRoutingLists/useRoutingLists.js

```diff
@@ -62,6 +62,6 @@ export const useRoutingLists = (poLineId) => {
 
   return ({
     routingLists: data || DEFAULT_DATA,
-    isLoading,
+    isLoading: isFetching,
```

Comment:
`isLoading` and `isFetching` have a different meaning, don't replace one with the other. Either provide both or use appropriate name